package.json

```diff
@@ -524,9 +524,9 @@
     },
     {
       "author": "stebalien",
-      "hash": "QmZXeeaAaSUCCizUfvZHK4BM8po7xJHFGodfYAQuDfHvwt",
+      "hash": "QmQYkj73ieX5qYTvY2H2fLhfh48FtUZhBcQVJRYgd3ZRnh",
       "name": "go-libp2p-pubsub-router",
-      "version": "0.3.9"
+      "version": "0.3.10"
```

Comment:
FYI, I had to re-publish so this'll have to be re-updated.

Reply:
ok, will do.

Reply:
Unfortunately this pulled a bunch of new deps, so lots of things need to be updated recursively... sigh

Reply:
Ok, this update touches too many things -- we'll have to wait for #5375 I think, as there are also code changes involved.

Reply:
I'll wait to update `go-libp2p-pubsub-router` until that pr has been merged.